modules/tinymce/src/core/main/ts/api/Settings.ts

```diff
@@ -167,10 +167,12 @@ const shouldDisableNodeChange = (editor: Editor): boolean => editor.getParam('di
 
 const isReadOnly = (editor: Editor): boolean => editor.getParam('readonly');
 
-const hasContentCssCors = (editor: Editor): boolean => getForcedRootBlock(editor) !== '';;
+const hasContentCssCors = (editor: Editor): boolean => editor.getParam('contentCssCors') !== '';
```

Comment:
This is still incorrect, as the setting is `content_css_cors`, not `contentCssCors`.